mne/io/neuralynx/tests/test_neuralynx.py

```diff
@@ -136,5 +171,52 @@ def test_neuralynx():
     )  # data
 
     _test_raw_reader(
-        read_raw_neuralynx, fname=testing_path, exclude_fname_patterns=["*u*.ncs"]
+        read_raw_neuralynx,
+        fname=testing_path,
+        exclude_fname_patterns=fname_patterns,
+    )
+
+
+@requires_testing_data
+def test_neuralynx_gaps():
+    """Test gap detection."""
+    # ignore files with no gaps
```

Comment:
This is one problem
```suggestion
    pytest.importorskip("neo")

    # ignore files with no gaps
```
but a worse problem is that it seems like `neo` is not being installed on any of those CIs. I'll push a commit